src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands.AnalysisServices.csproj

```diff
@@ -70,6 +70,9 @@
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="StartupScripts\*.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Mor module that don't yet have this, I would comment this out so we don't have an unnecessary dot-include in the psm1.